### PR TITLE
Updated delete /article_series/:id to use Zod validation

### PR DESCRIPTION
**Pull request by mixamum**

### CODE REVIEW

File: packages/server/src/api/index.ts [hunk 1]
```diff
@@ -576,14 +576,13 @@ router.put('/user/article_series/:id', async (req, res) => {
...
 router.delete('/user/article_series/:id', async (req, res) => {
-    if (!('id' in req.params)) {
-        res.status(400).send({ error: 'Id is required' });
-        return;
-    }
 
-    if (!req.params.id) {
-        // id cannot be 0
-        res.status(400).send({ error: 'Id is required' });
+    const idFromParams = z.coerce.number().int().positive();
```
Comment (prasenjeet-symon) [inline packages/server/src/api/index.ts:580]: There is already `id zod validator` try to use that and all the zod-related validation logic should reside in `validators` folder.

**Review by prasenjeet-symon — CHANGES_REQUESTED** [review]
Use correct folder and reuse the logic

**Comment by mixamum** [comment]
Ok

**Review by prasenjeet-symon — APPROVED** [review]
Awesome 👍🎉

**Pull request merged**